### PR TITLE
Upgrade updates

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/upgrade-cluster.adoc [hunk 1]
```diff
@@ -167,11 +166,16 @@ sudo ./gravity plan
...
-.. Example:  `--phase=/gc/rtf-controller-1`
-. Wait for the command to run and output an error again. If it does not output an error again, resume the upgrade by running the following command: `sudo ./gravity upgrade --resume`
-. If the command terminates with an error, read the logs to identify which node requires repair.
-.. Submit a ticket to MuleSoft support if assistance is required.
-. Open another terminal to the Runtime Fabric node identified in the error.
+For example,  `--phase=/gc/rtf-controller-1`
+. Wait for the command to run again. If the command does not terminate with an error, resume the upgrade by running the following command:
++
+----
+`sudo ./gravity upgrade --resume`
```
Comment (sundippatel) [inline modules/ROOT/pages/upgrade-cluster.adoc]: Remove the first and last ticks

**Review by sundippatel — DISMISSED** [review]
Looks great! One minor change requested.

File: modules/ROOT/pages/upgrade-cluster.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-cluster.adoc]: Got it - thank you!

File: modules/ROOT/pages/upgrade-cluster.adoc [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
 
+* Run the upgrade procedure on a non-production Runtime Fabric to verify compatibility with your environment and to gain familiarity with the procedure.
+* Plan the upgrade procedure when your team does not need to deploy new applications or managing existing applications on Runtime Fabric.
 
-* Run the upgrade procedure on a non-production Runtime Fabric first to verify compatibility with your environment and to gain familiarity with the procedure.
-* Run the upgrade procedure when your team does not need to deploy new applications or managing existing applications on Runtime Fabric.
-+
-The upgrade can take over an hour to complete.
+** The upgrade can take over an hour to complete.
+** The connection between Runtime Fabric and Anypoint Runtime Manager might become disconnected multiple times during the upgrade procedure. Connectivity will be restored after the upgrade has completed.
```
Comment (hannanelson) [inline modules/ROOT/pages/upgrade-cluster.adoc]: new style guidance says "no" to modal verbs: https://confluence.internal.salesforce.com/pages/releaseview.action?spaceKey=DOCTEAM&title=modal+verbs

Comment (hannanelson) [inline modules/ROOT/pages/upgrade-cluster.adoc]:
```suggestion
** The connection between Runtime Fabric and Anypoint Runtime Manager might become disconnected multiple times during the upgrade procedure. Connectivity is restored after the upgrade completes.
```

File: modules/ROOT/pages/upgrade-cluster.adoc [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
+* Make sure the `/tmp` directory has a minimum of 4 GiB of disk space available.
+* Verify you have root privileges. Root privileges are required to perform an upgrade.
+
+=== Environment Prerequisites
+
+* A production configuration of Runtime Fabric, with a minimum of 3 controller VMs, is required.
+* The internal load balancer must run on at least 3 replicas to maintain availability.
+* An external load balancer must be configured to load balance incoming requests to the controller VMs, with a health check configured for TCP port 443.
+* At least one worker node’s worth of CPU and memory resources must be available in the cluster.
```
Comment (hannanelson) [inline modules/ROOT/pages/upgrade-cluster.adoc:20]: Is there a way to reword?

File: modules/ROOT/pages/upgrade-cluster.adoc [hunk 4]
```diff
@@ -1,18 +1,28 @@
...
 +
-The connection between Runtime Fabric and Anypoint Runtime Manager might become disconnected multiple times during the upgrade procedure. Connectivity will be restored after the upgrade has completed.
-* The `/tmp` directory must have minimum of 4 GiB of disk space available.
-* Root privileges are required to perform the upgrade.
+This enables safe removal of a worker node from the cluster to apply upgrades, without impacting availability of applications.
+* Applications serving inbound requests must be scaled to a minimum of 2 replicas.
+
+[WARNING] Runtime Fabrics running on cluster version 1.0.x encounter downtime to applications if configured to use an HTTP proxy. An additional step in the procedure is required to apply the HTTP proxy configuration on each node after the upgrade. The Runtime Fabric version can be found by running `gravity status` on a node and referencing the cluster version.
```
Comment (hannanelson) [inline modules/ROOT/pages/upgrade-cluster.adoc]: any way to reword?

Comment (hannanelson) [inline modules/ROOT/pages/upgrade-cluster.adoc]: Reword to be active voice?

File: modules/ROOT/pages/upgrade-cluster.adoc [hunk 5]
```diff
@@ -167,11 +166,16 @@ sudo ./gravity plan
 sudo ./gravity upgrade --phase=< insert phase > --force --debug
 ----
 +
-.. Example:  `--phase=/gc/rtf-controller-1`
-. Wait for the command to run and output an error again. If it does not output an error again, resume the upgrade by running the following command: `sudo ./gravity upgrade --resume`
-. If the command terminates with an error, read the logs to identify which node requires repair.
-.. Submit a ticket to MuleSoft support if assistance is required.
-. Open another terminal to the Runtime Fabric node identified in the error.
+For example,  `--phase=/gc/rtf-controller-1`
```
Comment (hannanelson) [inline modules/ROOT/pages/upgrade-cluster.adoc]: terminating punctuation?

File: modules/ROOT/pages/upgrade-cluster.adoc [hunk 6]
```diff
@@ -184,7 +188,7 @@ sudo gravity plan --repair
 sudo ./gravity plan execute --phase=< insert phase > --force --debug
 ----
 
-The command should complete successfully. If it does not, wait a few minutes and repeat the previous steps.
+The command should complete successfully. If an error is returned, wait a few minutes and repeat the previous steps.
```
Comment (hannanelson) [inline modules/ROOT/pages/upgrade-cluster.adoc]: modal :)

**Review by hannanelson — APPROVED** [review]
Some wee editorial comments.

File: modules/ROOT/pages/upgrade-cluster.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-cluster.adoc]: Fixed - thank you!

Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-cluster.adoc]: Got it - thank you!

File: modules/ROOT/pages/upgrade-cluster.adoc (hunk 3 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-cluster.adoc:20]: Leaving for now.

File: modules/ROOT/pages/upgrade-cluster.adoc (hunk 4 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-cluster.adoc]: Leaving for now.

Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-cluster.adoc]: Reworded a bit - thank you!

File: modules/ROOT/pages/upgrade-cluster.adoc (hunk 5 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-cluster.adoc]: Added that - thank you!

File: modules/ROOT/pages/upgrade-cluster.adoc (hunk 6 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-cluster.adoc]: Fixed - thank you!

**Pull request merged**